tests/end-to-end/backup/pkg/tests/eventmesh/subscriber.go

```diff
@@ -0,0 +1,115 @@
+package eventmesh
+
+import (
+	"fmt"
+
+	"github.com/avast/retry-go"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// TODO(k15r): change image to a release image, not PR-XXX image
+const subscriberImage = "eu.gcr.io/kyma-project/pr/event-bus-e2e-subscriber:PR-4893"
```

Comment:
Cant we pass the image in a different way

Reply:
this image is just used to verify the events that will be sent during the test. honestly i would like to stick to this image.

Reply:
atm there is no point in making this one configurable, as the logic of the test highly depends on this image. It needs to adhere to some interface. 
I agree that we should probably rebuild this image and then use its sha, instead of the PR-TAG, but, this is then part of a followup ticket.

currently the same image is also used in the event bus subtest.